bin/node/node/src/chain_spec.rs

```diff
@@ -0,0 +1,150 @@
+use sp_core::{Pair, Public, sr25519};
+use bridge_node_runtime::{
+	AccountId, AuraConfig, BalancesConfig, GenesisConfig, GrandpaConfig,
+	SudoConfig, SystemConfig, WASM_BINARY, Signature
+};
+use sp_consensus_aura::sr25519::{AuthorityId as AuraId};
+use grandpa_primitives::{AuthorityId as GrandpaId};
+use sc_service;
+use sp_runtime::traits::{Verify, IdentifyAccount};
+
+// Note this is the URL for the telemetry server
+//const STAGING_TELEMETRY_URL: &str = "wss://telemetry.polkadot.io/submit/";
+
+/// Specialized `ChainSpec`. This is a specialization of the general Substrate ChainSpec type.
+pub type ChainSpec = sc_service::ChainSpec<GenesisConfig>;
+
+/// The chain specification option. This is expected to come in from the CLI and
+/// is little more than one of a number of alternatives which can easily be converted
+/// from a string (`--chain=...`) into a `ChainSpec`.
+#[derive(Clone, Debug)]
+pub enum Alternative {
+	/// Whatever the current runtime is, with just Alice as an auth.
+	Development,
+	/// Whatever the current runtime is, with simple Alice/Bob auths.
+	LocalTestnet,
```

Comment:
Do we really need both initially?

Reply:
Guess not, I'll remove the `LocalTestnet` variant